### DISCUSSION TITLE
How can I disable an autodiscovered repository

### DISCUSSION THREAD

**Discussion by tonnico** [renovatebot/renovate#20685] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.154.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I would like to disable an autodiscovered repository with a config file, e.g. renovate.json in the project itself.

### What you tried so far.

Add a renovate.json, without enabled managers.

```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "enabledManagers": []
}
```

But the dependencies are still discovered.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Branch summary (repository=test)
       "cacheModified": false,
       "baseBranches": [{"branchName": "main", "sha": "26ff598881f0a1d914e3029a4d2f3689267fe520"}],
       "branches": [
         {
           "automerge": false,
           "baseBranch": "main",
           "baseBranchSha": "26ff598881f0a1d914e3029a4d2f3689267fe520",
           "branchName": "renovate/pin-dependencies",
           "branchSha": "c3eedd58697cd8a1c5cc4cbf62b538ae62d31d09",
           "isModified": false,
           "isPristine": true
         },
         {
           "automerge": true,
           "baseBranch": "main",
           "baseBranchSha": "26ff598881f0a1d[914](https://.../renovate-bot/-/jobs/116726856#L914)e3029a4d2f3689267fe520",
           "branchName": "renovate/black-23.x",
           "branchSha": "6ff5a5a47cb07e56381432025248aae34a4fccc0",
           "isModified": false,
           "isPristine": true
         }
       ],
       "inactiveBranches": []
```

</details>

**Comment by viceice** [comment]
- https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter
- https://docs.renovatebot.com/self-hosted-configuration/#optimizefordisabled

**Comment by rarkins** [comment, marked as answer]
enabled=false

**Comment by tonnico** [comment]
Perfect, thanks for the very fast help.
